components/org.wso2.carbon.identity.local.auth.emailotp/src/main/java/org/wso2/carbon/identity/local/auth/emailotp/EmailOTPAuthenticator.java

```diff
@@ -240,6 +239,19 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
                 response, request, context);
     }
 
+    private AuthenticatedUser resolveUserFromUserStore(AuthenticatedUser authenticatedUserFromContext)
+            throws AuthenticationFailedException {
+
+        User user;
+        user = getUser(authenticatedUserFromContext);
```

Comment:
Can directly assign where the user variable is declared.

Reply:
Resolved by https://github.com/wso2-extensions/identity-local-auth-emailotp/pull/5/commits/fa9bd97219b16f6ab148004e38ac4c5a6b239c96